README.md

```diff
@@ -22,12 +22,16 @@ all, and write your own instead.
 
 ## Usage
 
+- [Plain JavaScript](#Button-(`superheroUtils.createButton`))
+- [React.js](#React.js-component)
```

Comment:
These links are not working on GitHub

Reply:
@shapkarin can you fix this?

Reply:
@davidyuk i fixed already and pushed, they are fine.